src/main/c/cext/ruby.c

```diff
@@ -4715,7 +4715,7 @@ void rb_define_virtual_variable(const char *name, VALUE (*getter)(ANYARGS), void
 }
 
 void rb_gc_register_mark_object(VALUE obj) {
-  rb_tr_error("rb_gc_register_mark_object not implemented");
+  RUBY_CEXT_INVOKE("rb_gc_register_mark_object", obj);
```

Comment:
Minor: this could be `RUBY_INVOKE_NO_WRAP()` to avoid wrapping the return value.

Reply:
Done.

Reply:
You mean `RUBY_CEXT_INVOKE_NO_WRAP` I think.

Reply:
Yes, indeed.